docs/user/unofficial-projects.rst

```diff
@@ -0,0 +1,98 @@
+Policy for Unofficial and Outdated Projects
+===========================================
+
+This policy describes a process where we take actions against unmaintained and unofficial forks of project documentation.
```

Comment:
Can the term "source project" be of use if it's defined?

```suggestion
This policy describes a process where we take actions against unmaintained and unofficial forks of project documentation.
Typically, these documentation projects describe a software project with versioned release cycles, but they can also be books or tutorials. Therefore, we refer to whatever source repository which the documentation project is connected to as the _source project_.
```

That can probably be shortened.